src/naive_bayes/bernoulli.rs

```diff
@@ -0,0 +1,278 @@
+use crate::error::Failed;
```

Comment:
I would suggest to add a documentation here to describe the general objectives of the module and sources

Reply:
I can help with documentation. I plan to cover most of the new functionality by a short summary, similar to [this one](https://github.com/smartcorelib/smartcore/blob/development/src/svm/svc.rs#L1) right before releasing v0.2.0. What would be helpful to me are links to resources that you used to learn about these methods either in description of the PR, issue or in the code somewhere. While I can (and will) use links to papers describing all NB methods in most of the cases there are other resources in the web that do a better job in explaining an algorithm and I like to share such resources with our users.